src/test/resources/pages/studentFeedbackSubmitPageFullyFilled.html

```diff
@@ -375,7 +375,7 @@
                         <textarea class="form-control" cols="100%" name="responsetext-4-0" rows="4">
                            Feedback to team 2.
                         </textarea>
-                        <input name="responseid-4-0" type="hidden" value="${question.id}%SFSubmitUiT.alice.b@gmail.tmt%Team 2"/>
+                        <input name="responseid-4-0" type="hidden" value="${question.id}%Team >'&quot;< 1%Team 2"/>
```

Comment:
This is correct. `"` gets a special treatment so that it can be parsed (otherwise, imagine parsing `value="abc"def"`). Everything else is shown as is, handled correctly or not.